ci/before_install.sh

```diff
@@ -1,10 +1,5 @@
-set -ex
-pushd ~
-
-# Workaround for Travis CI macOS bug (https://github.com/travis-ci/travis-ci/issues/6307)
-if [ "${TRAVIS_OS_NAME}" == "osx" ]; then
-    rvm get head || true
-fi
+#!/usr/bin/env bash
+set -exuo pipefail
```

Comment:
Seems like these broke all builds as we use `LD_LIBRARY_PATH` below without it being bound to anything.

Reply:
Yes, I pushed that right before bed and I have subsequently removed `-u` again.